scripts/exhafs_analysis.sh

```diff
@@ -441,9 +442,24 @@ ${NLN} ${COMINhafs_obs}/hafs.t${cyc}z.tldplr.tm00.bufr_d          tldplrbufr
 
 fi #USE_SELECT
 
+# Workflow will read from previous cycles for satbias predictors if ONLINE_SATBIAS is set to yes
+if [ ${ONLINE_SATBIAS} = "YES" ]; then
+  if [ ! -s ${COMhafsprior}/RESTART_analysis/satbias_hafs_out ] && [ ! -s ${COMhafsprior}/RESTART_analysis/satbias_hafs_pc.out ]; then
+    echo "Prior cycle satbias data does not exist. Grabbing satbias data from GDAS"
+    ${NLN} ${COMgfs}/gdas.$PDYprior/${hhprior}/${atmos}gdas.t${hhprior}z.abias           satbias_in
```

Comment:
Hi, Bin, not sure if this is the same on Hera, but on Orion, these bias data is stored in ${COMgfs}/gdas.$PDYprior/${hhprior}/atmos/gdas.t${hhprior}z.abias for my Laura tests.
Both for analysis & enkf

Reply:
@XL-OU This should be fine. The ${atmos} is "atmos/" or "" depending upon which version (GFSv15 or GFSV16) of the input GFS data is being used on all HAFS supported platforms.

Reply:
Hi, @BinLiu-NOAA, I see.  It was my bad, somehow the atmos is not defined in my exchange_analysis.sh & exchange_enkf.sh.  But I found them in the most up-to-date versions.  Thx.